css/style.css

```diff
@@ -1,7 +1,315 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+ }
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+html. body {
+	height: 100%;
+}
+
+body {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector "body" to come before selector "html. body" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455733)